.github/ISSUE_TEMPLATE/user_support.md

```diff
@@ -0,0 +1,12 @@
+---
+name: User Support
```

Comment:
I like adding this content.  Does this create a discussion or is it still an issue?  (Note sure what the "discussion" label does?  

Since we've added this can we remove the additions to the other issue templates?

Reply:
I removed them from the other ones, and created a label for Discussion. i am not sure how to check if creating this type of issue directs to discussion section or not.

Reply:
https://github.com/github/feedback/discussions/2858
Maybe this could provide some examples?